config/.credo.exs

```diff
@@ -41,10 +41,10 @@
         {Credo.Check.Refactor.UnlessWithElse},
         {Credo.Check.Warning.IExPry},
         {Credo.Check.Warning.IoInspect},
-        {Credo.Check.Warning.NameRedeclarationByAssignment},
-        {Credo.Check.Warning.NameRedeclarationByCase},
-        {Credo.Check.Warning.NameRedeclarationByDef},
-        {Credo.Check.Warning.NameRedeclarationByFn},
+        # {Credo.Check.Warning.NameRedeclarationByAssignment},
+        # {Credo.Check.Warning.NameRedeclarationByCase},
+        # {Credo.Check.Warning.NameRedeclarationByDef},
+        # {Credo.Check.Warning.NameRedeclarationByFn},
```

Comment:
What are we getting rid of here?

Reply:
None of these exist in credo anymore since they duplicate compiler functionality: https://github.com/rrrene/credo/issues/276